docs/rules/no-unused-expressions.md

```diff
@@ -34,6 +41,34 @@ f(0), {}
 a && b()
 ```
 
+If you enable the `allowShortCircuit` and/or `allowTernary` options when required you can use expressions like the following but keep in mind that the same restriction on expression side effects still applies:
+
+```js
+a && b()
+```
```

Comment:
We are currently adding configuration comments to the code blocks (see http://eslint.org/docs/rules/comma-dangle for an example).  By combing code blocks, it's only necessary to have one rule definition at the top of the block, rather than repeated in each block.  You're right that this style was used in some places in the docs, but unless there's a good reason to keep them separate, I've been going through and merging them together as I add the config comments.